docs/autogen/dest.py

```diff
@@ -29,12 +30,13 @@
     renamed to ``<args.ref>`` and is placed in ``<args.site>/docs/<args.ref>``, here are
```

Comment:
This line is not correct if we switch to `/en/`

Reply:
nice, catch. thanks.